packages/suite/src/views/wallet/coinmarket/common/CoinmarketLayout/CoinmarketLayoutNavigation.tsx

```diff
@@ -1,25 +1,59 @@
-import styled from 'styled-components';
-import { goto } from 'src/actions/suite/routerActions';
-import { useDevice, useDispatch, useSelector } from 'src/hooks/suite';
-import { WalletLayoutNavigation, WalletLayoutNavLink } from 'src/components/wallet';
+import styled, { css } from 'styled-components';
+import { useDevice, useSelector } from 'src/hooks/suite';
 import { getTitleForNetwork } from '@suite-common/wallet-utils';
 import { Translation } from 'src/components/suite';
 import { selectSelectedAccount } from 'src/reducers/wallet/selectedAccountReducer';
 import { FirmwareType } from '@trezor/connect';
+import { FONT_SIZE, FONT_WEIGHT } from '@trezor/components/src/config/variables';
+import { TranslationKey } from '@suite-common/intl-types';
+import { Route } from '@suite-common/suite-types';
+import { borders, spacingsPx } from '@trezor/theme';
+import { NavigationItem } from '../../../../../components/suite/Preloader/SuiteLayout/Sidebar/NavigationItem';
+import { IconName } from '@suite-common/icons';
+import { variables } from '@trezor/components';
 
-const SavingsWalletLayoutNavLinkWrapper = styled.div`
+const List = styled.div`
     display: flex;
-    margin-left: auto;
-    padding-left: 42px;
+    flex-direction: column;
+    width: 171px;
+    gap: ${spacingsPx.xxs};
+
+    ${variables.SCREEN_QUERY.BELOW_LAPTOP} {
+        width: 100%;
+        flex-direction: row;
+        overflow: scroll;
+
+        ::-webkit-scrollbar {
+            display: none;
+        }
```

Comment:
I don't think it's a good idea to hide scrollbar here:

<img width="512" alt="Screenshot 2024-01-25 at 13 02 42" src="https://github.com/trezor/trezor-suite/assets/858321/90ab372a-17d2-40ea-a975-bce211fb79ca">

Reply:
f6dba6eaa2